.github/workflows/nestjs.yml

```diff
@@ -0,0 +1,36 @@
+name: Nest.js ci/cd pipeline
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['master']
+
+jobs:
+  build:
+    runs-on: self-hosted
+
+    steps:
+      - uses: actions/checkout@v3
+      - run: docker stop $(docker ps -q --filter ancestor=dev-aurora-telegram )
+      - run: docker build -t dev-aurora-telegram .
+        if: always()
+      - name: send notification to aurora group
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: 'https://api.telegram.org/bot${{ secrets.TELEGRAM_TOKEN }}/sendMessage?chat_id=${{ secrets.TELEGRAM_TO }}&text=[success] aurora-telegram container ready'
```

Comment:
Please, add the repository name - you can get by looking for ${{github.repository}}